File: Euler2.java
public class Euler2
{
	public static void main(String[] args) {
	    long t1 = 1, t2 = 2, sum = 0, sumev = 0;
	    for (long i = 0; i < 100; i++) {
	        sum = t1 + t2;
	        t1 = t2;
	        t2 = sum;
	        if (t1 % 2 == 0) {
	            sumev += t1;
	        }
	        if (sum > 4000000) {
	            break;
	        }
	    }
	    System.out.println(t1);
	    System.out.println(sumev);
	}
	
	
}

File: Python/Euler2.py
num1 = 1
num2 = 2
sum = 0
sumev = 0
for i in range(0, 100):
    sum = num1 + num2
    num2 = sum
    num1 = num2
    if(num1%2==0):
        sumev = sumev + num1
    
    if(sum > 4000000):
        break

print(num1)
print(sumev)
    